nodejs/eks/cluster.ts

```diff
@@ -562,6 +563,22 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         });
     }
 
+    // Setup OIDC provider to leverage IAM roles for k8s service accounts.
+    let oidcProvider = undefined;
```

Comment:
I think this ends up being `any` - probably worth making the type explicit:

`let oidcProvider: aws.iam.OpenIdConnectProvider;`

Reply:
Done